app/models/alchemy/page/page_scopes.rb

```diff
@@ -36,7 +36,25 @@ module Page::PageScopes
 
       # All public pages
       #
-      scope :published, -> { where(public: true) }
+      scope :published, -> {
```

Comment:
Style/Lambda: Use the `lambda` method for multi-line lambdas. (https://github.com/bbatsov/ruby-style-guide#lambda-multi-line)